internal/pithosctl/pkg/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright (C) 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubernetes
+
+import (
+	"github.com/gravitational/trace"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// NewClient returns a new clientset for Kubernetes APIs
+func NewClient(kubeConfig string) (*kubernetes.Clientset, error) {
+	config, err := GetClientConfig(kubeConfig)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	return clientset, nil
+}
+
+// GetClientConfig returns client configuration,
+// if master is not specified, in-cluster configuration is assumed
```

Comment:
What does `master` mean in this context? From the implementation, it looks as if `kubeConfig` is empty, then in-cluster configuration is used.